package.json

```diff
@@ -378,6 +378,11 @@
             "require": "./dist/cjs/components/Progress/index.js",
             "module": "./dist/esm/components/Progress/index.js",
             "default": "./dist/esm/components/Progress/index.js"
+        },
```

Comment:
Well, we don't need add on this file too because we have a script to update this on the CI.

only required on exports file